### ISSUE TITLE
Add ECG ADaM data domains for QT

### ISSUE THREAD

**Issue by xni7** [SafetyGraphics/safetyData#8]
@jwildfire , there are some public ADaM data domains for the QT topic. 

see  https://physionet.org/content/ecgcipa/1.0.0/. The ADEG data look very useful sample data for the QT visualizations https://physionet.org/content/ecgcipa/1.0.0/adeg.csv

**Comment by jwildfire** [comment]
@xni7 Since the files are so large, I wonder if we should just put in a little "import_ecg()" function that calls `read_csv()` on the existing URLs and saves the data in your session (or alternatevely on disc). We'd roll right past the 5mb limit in CRAN otherwise.

**Comment by xni7** [comment]
I think it's definitely one option, @jwildfire . The risk might be the source url could be unreachable if the server is down, or the data is taken offline. I couldn't reach CTSpedia now to fetch Rich's QT sample data there. One option might be to download and host somewhere such as Github, and point to it. Another option might be to trim down the data by rows and/or columns. It's more work, and if we are already close to the 5mb mark, it may be difficult even if we trim

**Comment by jwildfire** [comment]
Good point. Maybe we can put the data in the repo under /dataraw but add it to rbuildignore
